lib/cli_opts.py

```diff
@@ -818,15 +822,33 @@ class CliOption(Option):
                         " format hypervisor:option=value,option=value,...",
                         default=[], action="append", type="identkeyval")
 
-NOIPCHECK_OPT = cli_option("--no-ip-check", dest="ip_check", default=True,
-                           action="store_false",
-                           help="Don't check that the instance's IP"
-                           " is alive")
+IPCHECK_OPT = cli_option("--ip-check", dest="ip_check", default=False,
+                           action="store_true",
+                           help="Check that the instance's IP is alive (ping)")
+
+def WarnDeprecatedOption(option, opt_str, value, parser):
```

Comment:
This should probably live in `lib/cli.py`.

Reply:
I have tried to move `WarnDeprecatedOption` to `lib/cli.py`, but ended up with an circular import error, because `lib/cli.py` also imports `cli_opts`:
```
# make
PYTHONPATH=. ./autotools/run-in-tempdir \
  /tmp/022a060c4e2e293f642f2a46006698f849e40834/saschalucas_colon_name_ip_check-tmp/./autotools/build-bash-completion --compact > doc/examples/bash_completion
Traceback (most recent call last):
  File "/tmp/022a060c4e2e293f642f2a46006698f849e40834/saschalucas_colon_name_ip_check-tmp/./autotools/build-bash-completion", line 50, in <module>
    from ganeti import cli
  File "/tmp/gntbuild.eKSZ4Lo2/ganeti/cli.py", line 58, in <module>
    import ganeti.cli_opts
  File "/tmp/gntbuild.eKSZ4Lo2/ganeti/cli_opts.py", line 831, in <module>
    action="callback", callback=cli.WarnDeprecatedOption,
                                ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: partially initialized module 'ganeti.cli' has no attribute 'WarnDeprecatedOption' (most likely due to a circular import)
make: *** [Makefile:4502: doc/examples/bash_completion] Error 1
make: *** Deleting file 'doc/examples/bash_completion'
```
Any hints?

Reply:
Probably in this case it's better to leave it here…